.github/workflows/php-syntax-errors.yml

```diff
@@ -0,0 +1,26 @@
+name: PHP Syntax Errors
```

Comment:
The Team51 scaffold is one single action right now that runs PHPCS with the **default** WP-Extra ruleset and some exceptions when those "suspicious" `vip_go` files. 

The PHPCS action in this PR doesn't rely on that 3rd-party setup routine but mimics the setup one would have on our computers + follows the rules of this repository. So it would also check for the correct language domain and for the correct prefixes, something which the current action doesn't support.

As for the PHP Syntax Errors action, it can help with making sure that our code is forward-compatible. And it makes sure that the code won't break the site if it wasn't tested on a dev site beforehand (as has happened with contractors that partners hired).

I'm happy if we use one, both, or none of them 😃 But I think they both have value on top of our existing action.